File: C#/C# 1/3. Operators-Expressions-Statements-Homework/Odd-Or-Even/OddOrEven.cs
/*Problem 1. Odd or Even  
  Write a program that reads an integer from the console, uses an expression to check if given integer is odd or even, and prints "even NUMBER" or "odd NUMBER", where you should print the input number's value instead of NUMBER.
  */

using System;

class OddOrEven
{
    static void Main()
    {
        // Read the input as a string (using ReadLine()) and convert this string to a number.
        int intNumber = int.Parse(Console.ReadLine());                                                       
        Console.WriteLine((intNumber % 2 == 0) 
            ? "even {0}" 
            : "odd {0}", intNumber);        
    }
}

File: Front-end/JS Fundamentals/operators and expressions/01. Odd or Even/odd-or-even.js
function solve(args) {
    var num = +args;
    if (num % 2) {
        console.log('odd ' + num);
    } else {
        console.log('even ' + num);
    }
}

solve("3");
solve("2");
solve("-2");
solve("-1");
solve("0");
